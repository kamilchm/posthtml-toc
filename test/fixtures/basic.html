<html>
<body>
<h1 id="title1">Title 1</h1>
<p>Text</p>
<h2 id="title2">Title 2</h2>
<p>Text</p>
<h3 id="title3">Title 3</h3>
<p>Text</p>
<h3 id="title3">Title 3</h3>
<p>Text</p>
<h2 id="title2">Title 2</h2>
<p>Text</p>
<h3 id="title3">Title 3</h3>
<p>Text</p>
<h4 id="title4">Title 4</h4>
<p>Text</p>
<h5 id="title5">Title 5</h5>
<p>Text</p>
<h5 id="title5">Title 5</h5>
<p>Text</p>
<h2 id="title2">Title 2</h2>
<p>Text</p>
</body>
</html>
