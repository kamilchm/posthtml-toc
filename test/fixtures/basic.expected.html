<html>
<body>
<h1 id="title1">Title 1</h1><div id="toc"><h2>Content</h2><ul><li><a href="#title2">Title 2</a><ul><li><a href="#title3">Title 3</a></li><li><a href="#title3">Title 3</a></li></ul></li><li><a href="#title2">Title 2</a><ul><li><a href="#title3">Title 3</a><ul><li><a href="#title4">Title 4</a><ul><li><a href="#title5">Title 5</a></li><li><a href="#title5">Title 5</a></li></ul></li></ul></li></ul></li><li><a href="#title2">Title 2</a></li></ul></div>
<p>Text</p>
<h2 id="title2">Title 2</h2>
<p>Text</p>
<h3 id="title3">Title 3</h3>
<p>Text</p>
<h3 id="title3">Title 3</h3>
<p>Text</p>
<h2 id="title2">Title 2</h2>
<p>Text</p>
<h3 id="title3">Title 3</h3>
<p>Text</p>
<h4 id="title4">Title 4</h4>
<p>Text</p>
<h5 id="title5">Title 5</h5>
<p>Text</p>
<h5 id="title5">Title 5</h5>
<p>Text</p>
<h2 id="title2">Title 2</h2>
<p>Text</p>
</body>
</html>
